docs/modules/ROOT/pages/component-prometheus.adoc

```diff
@@ -0,0 +1,25 @@
+= Prometheus
+
+[abstract]
+The Couchbase Prometheus Server is used to expose metrics and CMOS Prometheus server is used to scrape those metrics.
```

Comment:
The CMOS Prometheus instance can scrape any Prometheus endpoint, e.g. we are looking to provide node exporter or other additional tooling. I think it might be better to reword just to make this distinction: CMOS provides a Prometheus Server instance that can scrape any supported endpoint, Couchbase Server 7+ provides an endpoint to scrape.

That way it fits with usage of Prometheus exporter for other versions of Couchbase Server.